freqtrade/watchdog.py

```diff
@@ -0,0 +1,85 @@
+import os
+import signal
+import time
+import logging
+from multiprocessing import Value
+
+logger = logging.getLogger('freqtrade.watchdog')
+
+WATCHDOG_TIMEOUT = 300
+
+
+class Watchdog:
+
+    shared_heartbeat = Value('d', 0.0)
+    kill_signal = None
```

Comment:
As far as I can see this should be declared in the constructor and not as class variable